### ISSUE TITLE
Unable to know difference between blockquotes and indents

### ISSUE THREAD

**Issue by bbottema** [fraywing/textAngular#629]
I need to process TextAngular's output (the ng-model content variable), but I'm not able to see the difference between indents and blockquotes, as for both the editor uses blockquotes, nesting  inside a new blockquotes for each subsequent indent.

Currently the only way is to provide my own custom indent button with a different semantic markup, or by inspecting the HTML inserted into the DOM, as for indents the blockquote's left-border is made invisible.

**Comment by SimeonC** [comment]
I don't think this is something we can do anything about at the moment. It's too difficult to differentiate between them at a code level. Possibly you could modify the tools to add an extra class to the element when the blockquote tool is used.
